packages/react/src/AnchorNav/AnchorNav.stories.tsx

```diff
@@ -119,6 +119,7 @@ export const Playground = ({data, ...args}: {data: MockData; offset: number; sec
               <Text as="p">
                 AnchorNav is a component that allows users to navigate to different sections of a page.
               </Text>
+              <Button>Learn More</Button>
```

Comment:
@stamat Do you need the button here? Can we remove if not, or align to start of container?

Reply:
@rezrah I needed it to test the focus and see if it works correctly. I think it should be there for that sole purpose. But I'll align it to the start of the container! ❤️